### PR TITLE
Adding Week 6 Blogpost

### PR DESCRIPTION
**Pull request by ganimtron-10**
Adding Weekly Blogpost.

msedge_M5FpTfHo1P

### CODE REVIEW

**Review by m-agour — COMMENTED** [review]
Hi @ganimtron-10,
The name 'Week 6' makes me and @xtanion look so lazy :rofl:
Good job, there are no typos and it looks more like a blog post, mine looks more like a manifesto or something. 
Please see my suggestions below.

File: docs/source/posts/2022/2022-07-13-week-6-praneeth.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+Then I tried to group the shapes to transform and modify them all at once. For this, I had to investigate more about the key press and release events. Then I managed to select and deselect shapes by holding the ``Ctrl`` key.
+
+`Grouping Shapes <https://github.com/ganimtron-10/fury/tree/grouping-shapes>`_
+
+.. image:: https://user-images.githubusercontent.com/64432063/180261113-39760cba-0343-41e7-924a-c741eb838f0b.gif
+    :align: center
+    :width: 300
+
+I improved the `polyline` feature by adding a separate class to manage the creation and manipulation of the lines but still, I was facing the same issue with the dragging event, which initialized a new object every time a new line was created.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-07-13-week-6-praneeth.rst:29]: Maybe this better be `... the lines, but still; I was ...`

File: docs/source/posts/2022/2022-07-13-week-6-praneeth.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Did you get stuck anywhere?
+---------------------------
+Rotation from the center wasn't working because the shape wasn't pivoted at the lower left corner due to which it was hard to calculate the center using it. To handle this I created an interface between the actual pivot point and the center of the bounding box which made it easy to calculate and set the positions.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-07-13-week-6-praneeth.rst:33]: Missing a comma after `To handle this`

Comment (m-agour) [inline docs/source/posts/2022/2022-07-13-week-6-praneeth.rst:33]: Missing comma after `bounding box`

**Review by xtanion — COMMENTED** [review]
Hi @ganimtron-10, Nice bog. There's no typos or grammatical errors, good job 👍🏽

File: docs/source/posts/2022/2022-07-13-week-6-praneeth.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+.. image:: https://user-images.githubusercontent.com/64432063/180261113-39760cba-0343-41e7-924a-c741eb838f0b.gif
+    :align: center
+    :width: 300
+
+I improved the `polyline` feature by adding a separate class to manage the creation and manipulation of the lines but still; I was facing the same issue with the dragging event, which initialized a new object every time a new line was created.
+
+Did you get stuck anywhere?
+---------------------------
+Rotation from the center wasn't working because the shape wasn't pivoted at the lower left corner due to which it was hard to calculate the center using it. To handle this, I created an interface between the actual pivot point and the center of the bounding box, which made it easy to calculate and set the positions.
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-07-13-week-6-praneeth.rst:33]: > due to which it was hard to calculate the center using it

Not sure I understand, strange formulation.

**Review by skoudoro — CHANGES_REQUESTED** [review]
Hi @ganimtron-10,

Shorter are the sentences better and clearer it is.  Some of your sentences are really to understand. either, there are too long, or either you use `that` or `which` in a strange way.

Can you update some sentences? 

Thank you.

**Comment by ganimtron-10** [comment]
Thank You @skoudoro for the review. 
I will consider these suggestions in my future blogs and try to improve them.

**Comment by ganimtron-10** [comment]
Hello @skoudoro ,
Is there anything else left on this PR??
I have updated the post according to the required changes.
PTAL.
Thanks!

**Review by skoudoro — APPROVED** [review]
No, all good, merging

**Pull request merged**